lms/validation/authentication/_launch_params.py

```diff
@@ -46,20 +46,35 @@ class LaunchParamsAuthSchema(PyramidRequestSchema):
     def __init__(self, request):
         super().__init__(request)
         self._launch_verifier = request.find_service(name="launch_verifier")
+        self._application_instance_service = request.find_service(
+            name="application_instance"
+        )
 
     def lti_user(self):
         """
-        Return an models.LTIUser from the request's launch params.
+        Return a models.LTIUser from the request's launch params.
 
         :raise ValidationError: if the request isn't a valid LTI launch request
 
         :rtype: LTIUser
         """
         kwargs = self.parse(location="form")
 
+        try:
+            application_instance = (
+                self._application_instance_service.get_by_consumer_key(
+                    kwargs["oauth_consumer_key"]
+                )
+            )
+        except ApplicationInstanceNotFound as err:
+            raise marshmallow.ValidationError(
+                "Invalid OAuth 1 signature. Unknown consumer key."
+            ) from err
+
```

Comment:
I'm presuming this is going to branch into something with an `if` later on for 1.3?